feature-detects/forcetouch.js

```diff
@@ -0,0 +1,36 @@
+/*!
+{
+  "name": "Force Touch Events",
+  "property": "forcetouch",
+  "authors": ["Kraig Walker"],
+  "notes": [{
+    "name": "Responding to Force Touch Events from JavaScript",
+    "href": "https://developer.apple.com/library/prerelease/mac/documentation/AppleApplications/Conceptual/SafariJSProgTopics/Articles/RespondingtoForceTouchEventsfromJavaScript.html"
+  }]
+}
+!*/
+/* DOC
+Tests whether the browser supports the detection of Force Touch Events.
+Force Touch Events allow custom behaviours and interactions to take place based on the given pressure or change in pressure from a compatible trackpad.
+
+Force Touch events are available in OS X 10.11 and later on devices equipped with Force Touch trackpads.
+*/
+define(['Modernizr', 'hasEvent'], function(Modernizr, hasEvent) {
+  Modernizr.addTest('forcetouch', function() {
+
+    // Test if the browser supports the force touch event progression (see notes link)
+    if (!hasEvent('webkitmouseforcewillbegin', window) &&
```

Comment:
We should probably generate these event names via `Modernizr.prefixed` so in the case they become unprefixed.